Tests/Components/ApplePayDirect/Services/ApplePayPaymentMethodTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace MollieShopware\Tests\Components\ApplePayDirect\Services;
+
+use MollieShopware\Components\ApplePayDirect\Services\ApplePayPaymentMethod;
+use MollieShopware\Components\Services\PaymentMethodService;
+use PHPUnit\Framework\TestCase;
+
+class ApplePayPaymentMethodTest extends TestCase
+{
+    private $paymentMethodService;
+
+    public function setUp(): void
+    {
+        $this->paymentMethodService = new ApplePayPaymentMethod(
+            $this->createMock(PaymentMethodService::class)
+        );
+
+        require_once __DIR__ . '/../../../Shopware/Components/Plugin.php';
```

Comment:
why do we need this in here? shopware is a dev dependency?

Reply:
The constants in ShopwarePaymentMethod do refer to MollieShopware. This class does extend from Plugin. The kernel in the unit tests is not loaded correctly.

I haven't checked the pipeline, but when using the shopware release instead of the repository the '\Tests\bootstrap.php' is missing. For just having this one class the whole autoloading in the unit tests has to be changed.

Reply:
works for me in other screenshot